app/server/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+import {BeebopError} from "./errors/beebopError";
+import {APIResponse} from "./types/responseTypes";
+
+export function throwAPIError(response, error: any) {
+    if (error.response) {
+        const apiResponse = error.response.data.error as APIResponse<any>;
```

Comment:
There seems to be an extra top level `error` on the response sent back by `beebop_py` for some reason - should probably fix this in another ticket, as it's a bit confusing!